application/features/user/user.feature

```diff
@@ -0,0 +1,196 @@
+      | employee_id  | Employee ID  |
+      | first_name   | First Name   |
+      | last_name    | Last Name    |
+      | display_name | Display Name |
+      | username     | Username     |
+      | email        | Email        |
+
+  Scenario: Create a new user
+    Given a user does not exist with an employee_id of "123"
+      And the requester is authorized
+      And I provide the following valid data:
+        | property     | value                 |
+        | employee_id  | 123                   |
+        | first_name   | Shep                  |
+        | last_name    | Clark                 |
+        | display_name | Shep Clark            |
+        | username     | shep_clark            |
+        | email        | shep_clark@example.org|
+    When I request the user be created
+    Then the response status code should be 200
+      And the following data is returned:
+        | property     | value                 |
+        | employee_id  | 123                   |
+        | first_name   | Shep                  |
+        | last_name    | Clark                 |
+        | display_name | Shep Clark            |
+        | username     | shep_clark            |
+        | email        | shep_clark@example.org|
+        | active       | yes                   |
+        | locked       | no                    |
+      And last_changed_utc should be returned as now UTC
+      And last_synced_utc should be returned as now UTC
+      And the following data is not returned:
+        | property      |
+        | id            |
+        | password_hash |
+      And a user exists with an employee_id of "123"
+      And the following data should be stored:
+        | property     | value                 |
+        | first_name   | Shep                  |
+        | last_name    | Clark                 |
+        | display_name | Shep Clark            |
+        | username     | shep_clark            |
+        | email        | shep_clark@example.org|
+        | password_hash| NULL                  |
+        | active       | yes                   |
+        | locked       | no                    |
+      And last_changed_utc should be stored as now UTC
+      And last_synced_utc should be stored as now UTC
+
+  Scenario: "Touch" an existing user without making any changes
+    Given the requester is authorized
+      And I provide all the required fields
+      And I request the user be created
+      And a user exists with an employee_id of "123"
+    When I request the user be created again
+    Then a user exists with an employee_id of "123"
+      And the only property to change should be last_synced_utc
+      And last_synced_utc should be stored as now UTC
+
+  Scenario Outline: Change the properties of an existing user
+    Given the requester is authorized
+      And there are no users yet
+      And I provide all the required fields
+      And I request the user be created
+      And a user exists with an employee_id of "123"
```

Comment:
I'm not sure I understand the `Given`s (aka. state) here. It looks like there has to be both "no users" and "a user with a specific employee_id". I wonder if it would be worth refactoring this set of `Given`s a little to reduce that apparent contradiction.